daemon/container.c

```diff
@@ -305,6 +307,13 @@ container_new_internal(const uuid_t *uuid, const char *name, container_type_t ty
 	container->ram_limit = ram_limit;
 
 	/* Create submodules */
+	container->user = c_user_new(container, ns_usr);
+	if (!container->user) {
+		WARN("Could not initialize user subsystem for container %s (UUID: %s)",
+		     container->name, uuid_string(container->uuid));
+		goto error;
+	}
+
```

Comment:
init uid mapping here?

Reply:
no because its runtime depnended. Here just the container is created in state stopped.
We do this similar to the ip configuration in c_net at container start. after stopping a container the uid mapping is free for another container.